prometheus/summary.go

```diff
@@ -54,6 +54,9 @@ type Summary interface {
 }
 
 // DefObjectives are the default Summary quantile values.
+//
+// Deprecated: DefObjectives will be removed in v0.10. The default will be a
```

Comment:
Hm, it would be handy to still provide a package variable given how common these quantiles are. I doubt many people have a good feeling what error value to choose and would copy and paste it anyway.